lib/game.js

```diff
@@ -171,5 +173,15 @@ Game.prototype.instantiatePrefab = function(name) {
 	}.bind(this));
 	return id;
 };
-
-module.exports = Game;
+Game.prototype.registerPrefab = function (name,obj){
+	var prefabs = this.prefabs;
+	prefabs[name] = obj;
+};
+Game.prototype.importPrefabs = function(library){
```

Comment:
This should just be called `registerPrefabs` and the argument should be named `prefabs`